SimplePolicy/SimplePolicyPlugin.cs

```diff
@@ -93,7 +93,7 @@ void ILogPlugin.Log(string source, LogLevel level, string message)
         private bool VerifySizeLimits(Transaction tx)
         {
             // Not Allow free TX bigger than MaxFreeTransactionSize
-            if (tx.IsLowPriority && tx.Size > Settings.Default.MaxFreeTransactionSize) return false;
+            if (IsLowPriority(tx) && tx.Size > Settings.Default.MaxFreeTransactionSize) return false;
```

Comment:
We have a small problem here Shargon. ClaimTx cannot be paid (and they are often bigger than 1024... so users cannot pay nor cannot pass this limit), so I suggest we create another section called `CannotPayTxType` (including `ClaimTransaction`), so if transaction cannot be paid, this MaxFreeTransactionSize should not be imposed.

Reply:
We should be able to enable and disable this in the config, but yes